res/Sectors/M1105/Avereguar.txt

```diff
@@ -0,0 +1,339 @@
+2716                      E500000-0 Ba Va                        { -3 } (600+0) [0000] - -  - 004 12 HvFd M7 V           
+2717                      D745422-5 Ni Pa                        { -3 } (633-2) [4199] - -  - 822 11 HvFd M1 V K9 V      
+2718                      E110000-0 Ba                           { -3 } (800+1) [0000] - -  - 003  8 HvFd M1 V           
+2719                      AA7A515-E Ni Oc                        {  1 } (D42+2) [367A] - -  - 804 14 HvFd M5 V           
+2913                      E000000-0 As Ba Va                     { -3 } (400-4) [0000] - -  - 002 12 Na   G4 V M5 V      
+2918                      E100000-0 Ba Va                        { -3 } (A00-4) [0000] - -  - 004 15 Na   M1 V M5 V      
+3017                      D587834-5 Ri Ph Pa                     { -1 } (475+2) [B787] - -  - 820 12 Na   M0 V           
+3111                      C401404-A Ic Ni Va                     {  0 } (331-1) [5485] - N  - 900  3 Na   K9 V           
+3115                      D989301-7 Lo                           { -3 } (921-1) [1186] - N  - 820 11 Na   M0 V M3 V      
+3117                      D555854-6 Ph Pa                        { -2 } (478+1) [D656] - -  - 811  9 Na   F7 V M1 V      
+3213                      E223000-0 Ba Po                        { -3 } (700-3) [0000] - -  - 023 12 Na   M2 V M9 V      
+3214                      D330304-6 De Lo Po                     { -3 } (421-2) [1144] - -  - 304 11 Na   M9 V           
+0121                      E223120-7 Lo Po                        { -3 } (A01+2) [1152] - -  - 523 13 HvFd G5 V           
+0122 Kiakan               A665AW1-E Hi Ga Aqar5                  {  3 } (B9D-4) [DD5F] - E  - 624 10 HvFd M3 V           
+0124 Aqaroo               E321113-9 Lo Po He (Aqaroo) Pz         { -2 } (901+0) [514D] - -  A 803 11 HvFd M0 V           
+0125                      C100210-9 Lo Va AqarW                  { -1 } (711+3) [1188] - M  - 601 12 HvFd G8 V M3 V      
+0126                      C2446S5-8 Ag Ni                        { -1 } (855+0) [653B] - M  - 620 10 HvFd M6 V M3 V     
+0127                      A344731-D Ag Pi                        {  3 } (F66-4) [BA1F] - N  - 804 12 Na   M1 V           
+0223                      E300000-0 Ba Va                        { -3 } (900+1) [0000] - -  - 004  9 HvFd M6 V M8 V      
+0227                      B553626-8 Ni Po                        { -1 } (A54-2) [6587] - K  - 412 13 HvFd M8 V           
+0228                      B7537T1-8 Po                           {  1 } (669+0) [B86A] - KM - 910  9 HvFd G1 IV          
+0321                      C533220-6 Lo Po                        { -2 } (611+0) [6146] - M  - 504 14 HvFd G2 V       
+0325                      E310000-0 Ba                           { -3 } (400+1) [0000] - -  - 021 10 HvFd K8 V M4 V      
+0327 Itrasku              A7859T4-C Hi Pr Ga Cx                  {  3 } (B8D-3) [9C4F] - E  - 904 10 HvFd M6 V M8 V      
```

Comment:
0327 Itrask - This trade code should be `Cs` not `Cx`

Reply:
Done this, also added subsector capitals and cleanup a misplaced minor race